docs/specifications/cva6_requirement_specification.adoc

```diff
@@ -0,0 +1,391 @@
+
+|===
+|Config#|Target|32/64|FPU|MMU|L1 D$|L1 I$
+
+|cv32a6_imacf_sv32|FPGA|CV32A6|F|Sv32|32 kB|16 kB
+|cv32a6_imac_sv32|FPGA|CV32A6|None|Sv32|32 kB|16 kB
+|cv64a6_imacfd_sv39|ASIC|CV64A6|FD|Sv39|16 kB|16 kB
+|cv32a6_imac_sv0|ASIC|CV32A6|None|None (Bare mode)|None|4 kB
+|===
+
+== References
+
+=== Applicable specifications
+
+To ease the reading, the reference to these specifications can be implicit in the requirements below. For the sake of precision, the requirements identify the versions of RISC-V extensions from these specifications.
+
+[RVunpriv]   “The RISC-V Instruction Set Manual, Volume I: User-Level ISA, Document Version 20191213”, Editors Andrew Waterman and Krste Asanović, RISC-V Foundation, December 13, 2019.
+
+[RVpriv]  “The RISC-V Instruction Set Manual, Volume II: Privileged Architecture, Document Version  20211203”,  Editors  Andrew  Waterman, Krste  Asanović and John Hauser,  RISC-V Foundation, December 4, 2021.
+
+[RVdbg] “RISC-V External Debug Support, Document Version 0.13.2”, Editors Tim Newsome and Megan Wachs, RISC-V Foundation, March 22, 2019.
+
+[RVcompat] “RISC-V Architectural Compatibility Test Framework”, https://github.com/riscv-non-isa/riscv-arch-test[https://github.com/riscv-non-isa/riscv-arch-test]. +
+
+[AXI] AXI Specification, https://developer.arm.com/documentation/ihi0022/hc[https://developer.arm.com/documentation/ihi0022/hc]. +
+
+[CV-X-IF] Placeholder for the CV-X-IF coprocessor interface currently prepared at OpenHW Group; current version in https://docs.openhwgroup.org/projects/openhw-group-core-v-xif/[https://docs.openhwgroup.org/projects/openhw-group-core-v-xif/]. +
+
+[OpenPiton] “OpenPiton Microarchitecture Specification”, Princeton University, https://parallel.princeton.edu/openpiton/docs/micro_arch.pdf[https://parallel.princeton.edu/openpiton/docs/micro_arch.pdf]. +
+
+== Reference specifications
+
+[RVcmo] “RISC-V Base Cache Management Operation ISA Extensions, version 1.0-fd39d01, 2022-01-12” +
+
+[CLINT] Core-Local Interruptor (CLINT), “SiFive E31 Core Complex Manual v2p0”, chapter 6, https://static.dev.sifive.com/SiFive-E31-Manual-v2p0.pdf[https://static.dev.sifive.com/SiFive-E31-Manual-v2p0.pdf] + 
+
+== Other documents
+
+[FPGAreset] Ken Chapman, “Get Smart About Reset: Think Local, Not Global”, Xilinx WP272 white paper, https://www.xilinx.com/support/documentation/white_papers/wp272.pdf[https://www.xilinx.com/support/documentation/white_papers/wp272.pdf]. +
+
+== Functional requirements
+
+=== General requirement
+
+|===
+
+|GEN&#8209;10|CVA6 shall be *fully compliant with RISC-V specifications* [RVunpriv], [RVpriv] and [RVdbg] by implementing all mandatory features for the set of extensions that are selected and by passing [RVcompat] compatibility tests.
+
+|===
+
+As the RISC-V specification leaves space for options, this specification is a place where to specify these options.
```

Comment:
```suggestion
As the RISC-V specification leaves space for options, this specification specifies these options.
```
```suggestion
As the RISC-V specification leaves space for options, this specification is a place where to specify these options.
```

Reply:
Changed to "this specification specificies some of these variations"